### DISCUSSION TITLE
йой

### DISCUSSION THREAD

**Discussion by esisted** [PrivateBin/PrivateBin#949] [Q&A] [answered]

**Comment by r4sas** [comment]
???

**Reply by esisted** [reply]
the bot in the telegram does not respond to commands, what could be wrong?

**Reply by r4sas** [reply]
What does this have to do with PrivateBin?

**Reply by r4sas** [reply]
https://github.com/KyleJamesWalker/telebot

**Comment by rugk** [comment, marked as answer]
Locked as off-topic and not related to the project.
